stl/inc/thread

```diff
@@ -237,6 +242,11 @@ _NODISCARD inline bool operator==(thread::id _Left, thread::id _Right) noexcept
     return _Left._Id == _Right._Id;
 }
 
+#if _HAS_CXX20
+_NODISCARD inline strong_ordering operator<=>(thread::id _Left, thread::id _Right) noexcept {
+    return _Left._Id <=> _Right._Id;

```

Comment:
Fun fact: the Standard doesn't require the total ordering over thread ids that this function observes to be consistent with `operator==`, so we could have the two functions order them differently ;)